### DISCUSSION TITLE
Blocked on this page X (X%) confusion

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2886] [General] [answered]
Finally, after all these years, I'm beginning to understand why people don't understand what it means - https://www.reddit.com/r/uBlockOrigin/comments/17anksz/ublock_shows_a_perentage_of_trackers_blocked_on/

***

I asked people years ago what they think it means when they asked about it, and no one responded. And now over ~~20~~ 50 upvotes :roll_eyes:

**Comment by gorhill** [comment, marked as answer]
> uBlock shows a perentage of trackers blocked on the page

Nowhere does it say "Blocked trackers", his mind added this.

At most, it could be argued that it should be "Requests blocked on this page", but then it's starting to be a bit long for small screen, which I am sure is going to cause layout issues in some languages.

I say after all this years, let's not change this based on a few people who decided to assign meaning which is not there, and in case of doubt, simply looking at the documentation would have been faster than posting a new comment on reddit.
